src/Aspire.Dashboard/Components/Layout/MainLayout.razor.cs

```diff
@@ -87,12 +108,53 @@ public async Task LaunchSettings()
             Height = "auto"
         };
 
-        _ = await DialogService.ShowPanelAsync<SettingsDialog>(parameters).ConfigureAwait(true);
+        await DialogService.ShowPanelAsync<SettingsDialog>(parameters).ConfigureAwait(true);
+    }
+
+    public async Task OnPageKeyDownAsync(KeyboardEventArgsWithPressedKeys args)
+    {
+        if (args is { Key: "?", ShiftKey: true })
+        {
+            await LaunchHelpAsync();
+        }
+        else if (args.Key.ToLower() == "s" && args.ShiftKey)
+        {
+            await LaunchSettingsAsync();
+        }
+        else if (args.CurrentlyHeldKeys.Contains("g"))
```

Comment:
I realize this is done the same way that GitHub does it (both in terms of which key and how the keys are presented in the help dialog), but it took me a while to figure out that I was supposed to hold down the `g` key while pressing the second key, rather than doing it as a sequence. Maybe its just me, but that was not intuitive. @leslierichardson95  thoughts?

Reply:
Can we add a '+' or ',' in between the keys if they need to be pressed simultaneously or in sequence, respectively?  Also, are the + and - keys options for resizing panels instead of the left and right carets?  Personally, the former seems more intuitive but that said, I don't use keyboard shortcuts frequently in my day-to-day so take that with a grain of salt.

Reply:
There aren't any shortcuts whose keys need to pressed simultaneously, so I added + in the shortcuts. Changed the resizing keys as suggested 
![image](https://github.com/dotnet/aspire/assets/20359921/f87663f4-5977-4943-b05c-ba56edfa7a96)